### ISSUE TITLE
Is ASF development on laptop just unstable?

### ISSUE THREAD

**Issue by sathiathirumal** [Azure/service-fabric-issues#943]
I wonder if the constant sleeping and network switches is not handled well by the local cluster setup.
1) DNSService has well known problems with network interface switches
2) The local cluster becomes "unstable" after a few deploys and laptop going into sleep mode

e.g.
2>Started executing script 'GetApplicationExistence'.
2>Finished executing script 'GetApplicationExistence'.
2>Time elapsed: 00:00:16.1816665
2>Unable to determine whether the application is installed on the cluster or not

Once you get to this bad state, stopping a local cluster works usually. But (re) starting it fails 100%. A machine reboot is required to clear this up, which is an absolute pain as u can imagine.

![image](https://user-images.githubusercontent.com/9466367/37883406-fbf90762-305e-11e8-9957-ea1858843536.png)

![image](https://user-images.githubusercontent.com/9466367/37883416-0ba0a3b4-305f-11e8-89d3-a18ad245690d.png)

**Comment by sathiathirumal** [comment]
Not being able to kill fabricdnsservice.exe is the reason for needing the machine reboot. 

Task manager cannot kill the process. PSKILL cant either. ProcExp cant.

Any pointers to a tool that can terminate this process?

![image](https://user-images.githubusercontent.com/9466367/37883517-ccc6933c-305f-11e8-81e4-198595ffa0c3.png)

**Comment by masnider** [comment]
I've been able to kill it when running those tools as admin. Can you confirm if that is or isn't working for you?

**Comment by sathiathirumal** [comment]
@masnider yes, I can double-confirm that it doesnt work for me. This only happens AFTER I have gotten into the weird states as indicated by my prev responses + screenshots. When the local cluster is healthy, I can terminate that process just fine. Perhaps that gives a clue into why the process cannot be terminated?
